### PR TITLE
fix typo and outdated instructions in emp03

### PR DESCRIPTION
**Pull request by tothxa**

### CODE REVIEW

**Comment by Noordfrees** [comment]
LGTM :)

File: data/campaigns/emp03.wmf/scripting/texts.lua [hunk 1]
```diff
@@ -372,10 +372,10 @@ amalea_4 = {
...
       .. paragraphdivider() ..
       -- TRANSLATORS: Amalea
-      _([[After the production of enough wheat, we can dismantle the farm and build a sheep farm to produce wool. Later on, we could build a weaving mill in the same spot to use our wool for weaving cloth. But for the moment being, let’s start by producing a reasonable amount of wheat.]])
+      _([[After the production of enough wheat, we can dismantle the farm and build a sheep farm to produce wool. When the sheep farm starts working, we could find or free up space for a medium building to build a weaving mill to use our wool for weaving cloth. But for the moment being, let’s start by producing a reasonable amount of wheat.]])
       .. paragraphdivider() ..
       -- TRANSLATORS: Amalea
-      _([[I think a stock of 50 sheaves of wheat would be enough for starters. Perhaps we will have to adjust the target quantity for wheat to achieve this.]]))
+      _([[I think a stock of 50 sheaves of wheat would be enough for starters. We do not have a target quantity for wheat, because the farmer does not need other wares to produce it. All we have to do is watch our stock of wheat grow.]]))
```
Comment (hessenfarmer) [inline data/campaigns/emp03.wmf/scripting/texts.lua:378]: well, this one was in fact meant as a hoax to the player. However I admit that this is not very kind from me and could lead to some complaints in the forums.   
So I am happy with the chage but I would be happy as well with deleting the original second sentence conpletely.

**Review by hessenfarmer — APPROVED** [review]
Well spotted and fixed. One comment inline for explanation.

File: data/campaigns/emp03.wmf/scripting/texts.lua (hunk 1 above)
Comment (tothxa) [reply data/campaigns/emp03.wmf/scripting/texts.lua:378]: I thought about removing it, but as this is still kind of a teaching scenario, I thought some explanation wouldn't hurt.

Comment (hessenfarmer) [reply data/campaigns/emp03.wmf/scripting/texts.lua:378]: ok, I'll merge it then

**Pull request merged**